tests/stage_2/mod-2.txt

```diff
@@ -1,2 +1,2 @@
-brun -s2 '(opt (com (q (mod (N) (defun fact (N) (if (= N 1) 1 (* N (fact (- N 1))))) (fact N)))))'
-(q ((c (q ((c 2 (c 2 (c 5 (q ())))))) (c (q ((c (i (= 5 (q 1)) (q (q 1)) (q (* 5 ((c 2 (c 2 (c (- 5 (q 1)) (q ())))))))) 1))) 1))))
+brun -s2 '(opt (com (q . (mod (N) (defun fact (N) (if (= N 1) 1 (* N (fact (- N 1))))) (fact N)))))'
```

Comment:
all these need to be run with `-n` to print the results as numbers, rather than operators. I think richard is making that change.